tests/shared/features/shared/web_steps.feature

```diff
@@ -293,3 +293,8 @@ Feature: Web steps
       And the "Disabled field #3" field should be disabled
     But the "Enabled field #1" field should not be disabled
       And the "Enabled field #2" field should not be disabled
+
+  @javascript
+  Scenario: /^the window should be titled "([^"]*)"$/
+    When I go to "/static_pages/home"
+    Then the window should be titled "spreewald test application"
```

Comment:
I would like to have a second test without the javascript-tag. A basic functionality like this should not force the user to use a certain driver (and also I guess many people would not understand why the step is not working when they accidentially use a non-javascipt driver.)
I guess stripping the title before comparing should be enough to make it work.